app/authentication/authenticator.py

```diff
@@ -29,17 +29,23 @@ def user_loader(user_id: str) -> Optional[str]:
 
 
 @login_manager.request_loader
-def request_load_user(
-    request: Request,
-) -> Optional[User]:  # pylint: disable=unused-argument
+def request_load_user(request: Request) -> Optional[User]:
     logger.debug("load user")
-    return load_user()
+
+    if request.endpoint:
+        extend_session = not (
+            "session_expiry" in request.endpoint and request.method == "GET"
```

Comment:
I think this should be checked explicitly.
```suggestion
            "session.session_expiry" == request.endpoint and request.method == "GET"
```